tests/chunk_topography_postprocessor.prm

```diff
@@ -15,8 +15,9 @@ set Use years in output instead of seconds = true
 
 subsection Boundary temperature model
   set List of model names = constant
+  set Fixed temperature boundary indicators   = inner, outer
   subsection Constant
-    set Boundary indicator to temperature mappings = 0:0,1:0,2:0,3:0
+    set Boundary indicator to temperature mappings = inner:0,outer:0
```

Comment:
I guess what this means is that the input file previously specified temperatures for boundaries for which there is nothing to be specified, rigth?

Reply:
There's no functional change to this test; I've just changed the indicator numbers to their associated strings and removed the two boundaries that weren't listed as Fixed temperature boundary indicators.

Reply:
Right, that's what I meant: you removed indicators that the user (probably in error) specified.